js/transfer-button.js

```diff
@@ -1,20 +1,67 @@
 'use strict'
 
 var output = document.getElementById('greeter-output');
-output.innerHTML = 'Click the button to check temperature in Fahrenheit scale.' + '<br><br>' + output.innerHTML;
-
+output.innerHTML = 'Click the red button to check temperature in Fahrenheit scale.' + '<br><br>';
+var foutput = document.getElementById('greeter-foutput');
+foutput.innerHTML = 'Click the blue button to check temperature in Celcius scale.' + '<br><br>';
 var button = document.getElementById('greeter-button');
+var fbutton = document.getElementById('greeter-fbutton');
 var temperature;
 var Ftemperature;
+var regex = /^[0-9]+$/;
 
 button.addEventListener('click', function() {
     temperature = window.prompt('What\'s the temperature? Enter the number: ');
-    if (typeof temperature == 'number') {
-        output.innerHTML = 'You ddhave to ffenter the number!' + '<br><br>' + output.innerHTML;
+
+    if (!temperature.match(regex)) {
+        output.innerHTML = 'You have to enter the number!' + '<br><br>';
     }
     else {
         Ftemperature = (temperature * 1.8) + 32;
         output.innerHTML = 'Celsius temperature is ' + temperature + '°C' + '\n' +
-        ' and Fahrenheit temperature is ' + Ftemperature + '°F' + '<br><br>' + output.innerHTML;
+        ' and Fahrenheit temperature is ' + Ftemperature + '°F' + '<br><br>';
+        if (temperature == 0) {
+            output.innerHTML = 'This temperature is a melting point of ice.' + '<br><br>' + output.innerHTM;
+        }
+        else if (Ftemperature > 120) {
+            output.innerHTML = 'Most bacteria do not live above this temperature.' + '<br><br>' + output.innerHTML;
+        }
+        else if (temperature >= 20) {
```

Comment:
`20`? you need to know what this value is - read about Magic Numbers antipattern ;)